src/js/components/List/doc.js

```diff
@@ -3,7 +3,16 @@ import { describe, PropTypes } from 'react-desc';
 import { genericProps, getAvailableAtBadge } from '../../utils';
 
 const sizes = ['xxsmall', 'xsmall', 'small', 'medium', 'large', 'xlarge'];
-const sides = ['horizontal', 'vertical', 'top', 'bottom', 'left', 'right'];
+const sides = [
+  'horizontal',
+  'vertical',
+  'top',
+  'bottom',
+  'left',
+  'right',
+  'start',
+  'end',
```

Comment:
As a reader, it is not clear to me what is the difference between `side="start"` and `side="right"`.

Reply:
I think we'll eventually need a general/global documentation blurb on how dir layout works.